src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void printHorizontalLine() {
+        String horizontal_line = "____________________________________________________________";
+        System.out.println(horizontal_line);
+    }
+
+    public static void welcome() {
```

Comment:
```suggestion
    public static void printWelcomeMessage() {
```
Perhaps this function could be renamed to printWelcomeMessage so that it better describes what the function does?

Reply:
Thanks!